HelpSource/Help.schelp

```diff
@@ -2,38 +2,266 @@ TITLE::Help
 summary::Documentation home
 categories::Help
 
-SuperCollider is an audio server, programming language, and IDE for sound synthesis and algorithmic composition.
-
 NOTE:: link::Guides/News-3_13##News in SuperCollider version 3.13:: ::
 
-SECTION::Search and browse
+
+SECTION:: An introductory overview
+
+*SuperCollider* is a cross-platform environment for audio synthesis and algorithmic composition used by musicians, artists and researchers working with sound, with three main components:
+
 definitionlist::
-## link::Search:: || Search all documents and methods
-## link::Browse:: || Browse all documents by categories
+## soft:: ::• soft:: ::link::Classes/Server:: – a real-time audio engine.||
+The default server is link::Classes/Server#Switching the server application#scsynth::. To take full advantage of multi-core, an alternative sound engine, link::Classes/Server#Switching the server application#supernova::, can be used.
+
+## soft:: ::• soft:: ::Client soft:: ::– sclang. a text-based programming language and interpreter. ||
+
+emphasis::sclang:: controls audio synthesis processes on the server and offers an link::Overviews/JITLib##interactive programming environment for on-the-fly coding::, link::Tutorials/A-Practical-Guide/PG_01_Introduction##an extensive library for creating patterns:: and link::Classes/Scale##a library of scales and modes from various cultures::.
```

Comment:
```suggestion
emphasis::sclang:: controls audio synthesis processes on the server and offers an interactive programming environment for link::Overviews/JITLib##on-the-fly coding::, an extensive library for creating link::Tutorials/A-Practical-Guide/PG_01_Introduction##patterns:: and a library of link::Classes/Scale##scales and modes:: from various cultures.
```

Reply:
I'd also suggest "live coding" instead of "on-the-fly" coding, just because it's a bit more recognizable these days.

Reply:
> I'd also suggest "live coding" instead of "on-the-fly" coding, just because it's a bit more recognizable these days.

done!